projects/components/src/lib/advanced-search/advanced-search.scss

```diff
@@ -58,3 +55,10 @@ input {
 .menu-title {
   @include menu-title();
 }
+
+.new-filter {
+  float:  right;
```

Comment:
Float's have gone out of fashion lately due to the unexpected problems they tend tend to case. The best way to manage layout nowadays is with CSS Flexbox.

In this case, we can set `advanced-search` to be `display: flex` (not sure why it's being overriden as block actually), which lets us move the button to the right using `align-self: flex-end`.

Reply:
Instead of `align-self: flex-end`, I needed to use `justify-content: flex-end`. This is also causing an interesting echoed button effect.

<img width="494" alt="Screen Shot 2019-07-10 at 11 35 17 PM" src="https://user-images.githubusercontent.com/11209247/61027594-6dc58480-a36b-11e9-986c-45586b58a078.png">

Reply:
Oo weird, I'm not sure I've seen that before. Let me know if you need help figuring out why that happened